### PR TITLE
User experience enhancements

### PR DESCRIPTION
**Pull request by grs83**

### CODE REVIEW

**Comment by grs83** [comment]
screen shot 2018-04-06 at 2 02 27 pm
screen shot 2018-04-06 at 2 03 24 pm

File: app.js [hunk 1]
```diff
@@ -35,12 +35,13 @@ export default class App extends Component {
       showModal: false,
       modalItem: {},
       locationModal: false,
-      noResultsModal: false
+      noResultsModal: false,
+      searchTitle: ''
```
Comment (thebearingedge) [inline app.js]: Give this property a more meaningful name.

File: app.js [hunk 2]
```diff
@@ -191,10 +203,17 @@ export default class App extends Component {
   }
 
   render() {
+    const itemsTitle = this.state.oArgs.category
```
Comment (thebearingedge) [inline app.js]: If you can simplify this into a function, go for it!

File: app.js [hunk 3]
```diff
@@ -191,10 +203,17 @@ export default class App extends Component {
...
+      ? this.state.oArgs.category.toUpperCase()
+      : this.state.searchTitle
+        ? `Search For: ${this.state.searchTitle.toUpperCase()}`
+        : 'Searching All Categories';
     return (
       <div>
         <Location
-          welcomeMessage={this.state.welcomeMessage}
+          keyPressHander={e => {
+            e.charCode !== 13 ? null : this.setState({ locationModal: false });
```
Comment (thebearingedge) [inline app.js]: If you don't want to use an if for some reason, use `&&` or `||`.

File: app.js [hunk 4]
```diff
@@ -204,11 +223,15 @@ export default class App extends Component {
           noResultsModal={this.state.noResultsModal}
         />
         <SearchBar
+          keyPressHandler={e => {
+            e.charCode !== 13 ? null : this.clickHandlerSearchBar();
```
Comment (thebearingedge) [inline app.js]: Samesees.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Take a look at your alignment of your layout. Especially the search results.

**Comment by grs83** [comment]
screen shot 2018-04-06 at 2 48 28 pm

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**